tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -298,20 +310,24 @@ def __init__(self, test_ctx, *args, **kwargs):
                 'kafka_connection_rate_limit': 10000,
                 'kafka_connections_max': 50000,
 
-                # Enable segment size jitter as this is a stress test and does not
-                # rely on exact segment counts.
-                'log_segment_size_jitter_percent': 5,
-
                 # In testing tiered storage, we care about creating as many
                 # cloud segments as possible. To that end, bounding the segment
                 # size isn't productive.
                 'cloud_storage_segment_size_min': 1,
                 'log_segment_size_min': 1024,
 
+                # Use more connections in testing to reach reasonable throughput
+                # number even for small segments and high S3 latency.
+                'cloud_storage_max_connections': 100,
+
                 # Disable segment merging: when we create many small segments
                 # to pad out tiered storage metadata, we don't want them to
                 # get merged together.
                 'cloud_storage_enable_segment_merging': False,
+
+                # Disable scrubbing for the duration of the test to reduce S3 request rate.
```

Comment:
The test should be as realistic as possible what is the problem with high S3 request rate? If the scrubber struggles in any form with the high partition count then it should really be enabled.

Reply:
The problem is actually not the request rate. The scrubber isn't even triggered for most of the test as the default interval is 1h. Updating the comment to make the reason a bit more clear.